docs/conf.py

```diff
@@ -25,6 +25,7 @@
 
 linkcheck_ignore = [
     'https://www.iso.org/obp/ui/.*',  # Breaks accessibility via JS ¯\_(ツ)_/¯
+    'https://dev.mysql.com/.*',  # 403 Client Error: Forbidden for url
```

Comment:
which link is failing? should we try to replace it instead of blacklisting it here?

Reply:
I would then ask you to remove it, please.
If it fails regularly we might need to find another url to replace it.

Reply:
I think the errors are originating from protection mechanisms against rogue crawlers intending to feed machine learning models, that's why database vendors are impacted particularly, but also many others.

Those links work perfectly well when requested by humans, only on machines, they fail. In this spirit, I think using the `linkcheck_ignore` setting is the right choice to make the linkchecker skip those.

Please let me know if you think different.